packages/wells/src/client/api/wellbores.ts

```diff
@@ -0,0 +1,118 @@
+//import { SpatialRel, GeometryRel } from '@cognite/geospatial-sdk-js';
+import { Asset, AssetsAPI } from '@cognite/sdk';
+import { SearchWellbores, Wellbore } from '../model/Wellbore';
+//import { WellHeadLocation } from '../model/WellHeadLocation';
+//import { geospatialClient } from './utils';
+//import { GeoJson } from '../model/GeoJson';
+
+export class Wellbores extends AssetsAPI {
+  /**
+   * converts asset into a wellbore
+   *
+   * ```js
+   * var wells: WellHeadLocation = Wellbores.mapToWellHeadLocation(asset);
+   * ```
+   *
+   * @param asset
+   */
+  static mapToWellbore = (assets: Asset[]): Wellbore[] => {
+    return assets.map(asset => {
+      return <Wellbore>{
+        id: asset.id,
+        name: asset.name,
+        parentId: asset.parentId,
+        metadata: asset.metadata,
+      };
+    });
+  };
+
+  /**
+   * A generic template for searching wellbores based on strict filtering and/or fuzzy filtering
+   *
+   * ```js
+   * const created = await client.wellbores.searchAssets(exactSearch: {key: val}, fuzzySearch: {key: val});
+   * ```
+   *
+   * @param exactSearch Filter on assets with strict matching.
+   * @param fuzzySearch Fulltext search for assets. Primarily meant for for human-centric use-cases, not for programs.
+   */
+  private searchAssets = async (
+    exactSearch = {},
+    fuzzySearch = {}
+  ): Promise<Asset[]> => {
+    const body = {
+      filter: {
+        metadata: {
+          type: 'Wellbore',
+        },
+        ...exactSearch,
+      },
+      search: {
+        ...fuzzySearch,
```

Comment:
I don't think the fuzzy search is being used in this class? I guess it might be used in the future, but personally I would remove it for now and then `this.search` could then be exchanged for `this.list`.

Reply:
Removed the fussySearch part. `this.list` return `CursorAndAsyncIterator<Asset>`, so i'd rather use `this.search` which returns an `Promise<Asset[]>`.